fiftyone/utils/eval/detection.py

```
@@ -610,6 +638,10 @@ def _parse_config(pred_field, gt_field, method, is_temporal, **kwargs):
         else:
             method = fo.evaluation_config.default_detection_backend
 
+    custom_metrics = kwargs.get("custom_metrics", None)
```

Ruff (0.8.2)

Use `kwargs.get("custom_metrics")` instead of `kwargs.get("custom_metrics", None)`

Replace `kwargs.get("custom_metrics", None)` with `kwargs.get("custom_metrics")`

(SIM910)